dromedary.py

```diff
@@ -163,6 +180,166 @@ def print_output(out: subprocess.CompletedProcess) -> None:
     print(std_out + err_out)
 
 
+###############################################################################
+def run_cmd_output(cmd_args: List[Any], cmd_env: Optional[Dict[str, str]]) -> str:
```

Comment:
Why is `cmd_args` a `List[Any]` instead of `List[str]` ?

Reply:
I have to put quotes (because of whitespace in paths and e.g. a package may contain a "<" or ">") around each argument which implicitly converts (well, calls `str()`) each argument to a string. This just makes that explicit. If you prefer a `List[str]`, I can of course change that. Or I could add a  [Protocol](https://docs.python.org/3/library/typing.html#typing.Protocol)  which made that explicit and use that instead of `Any`.